docs/api/field_types_reference/measurementfield.md

```diff
@@ -0,0 +1,135 @@
+The service contains the following API endpoints:
+- `buildSimpleValue` that is used to handle a single value
+- `buildRangeValue` that is used to handle a range
+
+Assuming that the service exists as `$measurementService`, the expected input 
+examples are as follows:
+
+| Type                                                    | Example                                                             |
+|---------------------------------------------------------|---------------------------------------------------------------------|
+|`\Ibexa\Contracts\Measurement\Value\SimpleValueInterface`| `$measurementService->buildSimpleValue('length', 2.5, 'centimeter')`|
+|`\Ibexa\Contracts\Measurement\Value\RangeValueInterface` | `$measurementService->buildRangeValue('length', 1.2, 4.5,  'inch')` |
+
+### Value object
+
+##### Properties
+
+The Value class of this Field Type contains the following properties:
+
+| Property | Type                                             | Description                                                                                                          |
+|----------|--------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `$value` |`Ibexa\Contracts\Measurement\Value\ValueInterface`| Stores the Measurement API Value, which can be either an instance of `Ibexa\Contracts\Measurement\Value\SimpleValueInterface` or `Ibexa\Contracts\Measurement\Value\RangeValueInterface`. |
+
+##### Constructor
+
+The `Measurement\Value`constructor for this Value object initializes a new Value 
+object with the value provided. 
+As its first argument it accepts an object of `Ibexa\Contracts\Measurement\Value\ValueInterface` type.
+
+Depending on the selected input type, the object resembles the following examples:
+
+``` php
+// Simple input (single value) example
+
+// @var MeasurementServiceInterface $measurementService
+
+// Instantiates a Measurement Value object
+$measurementValue = new Measurement\Value(
+    $measurementService->buildSimpleValue(
+                    'length',
+                    13.5,
+                    'centimeter'
+    )
+);
+```
+
+``` php
+// Range input value example
+
+// @var MeasurementServiceInterface $measurementService
+
+// Instantiates a Measurement Value object
+$measurementValue = new Measurement\Value(
+    $measurementService->buildRangeValue(
+        'volume',
+        0.5,
+        0.7,
+        'liter'
+    )
+);
+```
+
+### Validation
+
+The Measurement Field Type validates measurement types and units passed within 
+the Value object against a list of the ones that the system supports, which can 
+be found in the `vendor/ibexa/measurement/src/bundle/Resources/config/builtin_units.yaml` file.
+
+### Extending the default list of Measurement types and units
```

Comment:
I think this would work best as a separate topic, maybe somewhere alongside https://doc.ibexa.co/en/master/extending/add_user_setting/

Reply:
Yes, I considered placing this information in a separate article but Andrew insists we keep it here, as it will most likely be invalidated by features that he is currently working on.